mayavi/core/scene.py

```diff
@@ -58,6 +58,9 @@ class Scene(Base):
         'mayavi.core.mouse_pick_dispatcher.MousePickDispatcher',
         record=False)
 
+    # The scene has been closed.
+    closed = Event()
```

Comment:
It looks that this event is not used anywhere. It should be removed then to avoid confusion